### ISSUE TITLE
[Request] RDM - Change Melee Capstone Combo Button to Prefulgence when "Prefulgence Ready" is available. 

### ISSUE THREAD

**Issue by Zynyx** [MKhayle/XIVComboExpanded#264]
TLDR; Make the Melee combo button change to Prefulgence whenever the RDM is under the effect of "Prefulgence Ready."

Few notes on this one; first, Manafication has changed from granting RDM's 50 of each color of mana upon use to instead granting 6 stacks of free Melee Combo uses that allow one to complete a full melee combo (Slightly different way of achieving the same effet). 

In addition, the new RDM LVL 100 ability Prefulgence operates by applying a 30 second buff to Red Mages upon completely using all of their 6 Manafication stacks which is called "Prefulgence Ready". While "Prefulgence Ready" is up, Manafication gets switched to "Prefulgence" allowing for its use; however, this is muddied by the already existing XIVCombo shortcut of changing Embolden to Manafication when available. Once manafication is used, you are no longer able to access it again until the CD comes up making it impossible to use Prefulgence without assigning Manafication to a new key. 

Long story short, the easy fix for this would be to make the Melee combo button change to Prefulgence whenever the RDM is under the effect of "Prefulgence Ready."

The RDM's melee button should already by on Riposte (or Potentially Enchanted Riposte depending on your current mana level) by the end of a proper Manafication cycle anyway.

**Comment by MKhayle** [comment]
I'll look at RDM tomorrow and try to figure out how to manage it properly

**Comment by MKhayle** [comment]
I'm not good enough at black magicks but I guess my white magicks knowledge compensate it
should be good with 1.5.0.17

**Comment by Zynyx** [comment]
That did the trick; Works a charm. Thank you very much!
